libs/srt/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2024 Koen Vandeputte <koen.vandeputte@citymesh.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=srt
+PKG_VERSION:=1.5.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/haivision/srt/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=befaeb16f628c46387b898df02bc6fba84868e86a6f6d8294755375b9932d777
+PKG_MAINTAINER:=Koen Vandeputte <koen.vandeputte@citymesh.com>
+
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_INSTALL:=1
```

Comment:
Already default.

Reply:
removed.
A lot of packages still carry this around.  I smell a cleanup action :-)